src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -148,6 +158,16 @@ class BlockchainImpl(
                       protected val totalDifficultyStorage: TotalDifficultyStorage
                     ) extends Blockchain {
 
+
+  private val byteArrayUInt256Serializer = new ByteArrayEncoder[UInt256] {
```

Comment:
The following two are also implemented here https://github.com/input-output-hk/etc-client/blob/phase/4/jsonrpc/src/main/scala/io/iohk/ethereum/ledger/InMemoryWorldStateProxy.scala#L17

Maybe it should be extracted to a common place, what do you think?